examples/ridge_reg_implicit_diff.py

```diff
@@ -43,19 +43,19 @@ def matvec(u):
 
   return linear_solve.solve_cg(matvec=matvec,
                                b=jnp.dot(X_tr.T, y_tr),
-                               ridge=len(y_tr) * lam,
+                               ridge=len(y_tr) * l2reg,
                                x0=init_params,
                                maxiter=20)
 
 
 # Perhaps confusingly, theta is a parameter of the outer objective,
-# but lam = jnp.exp(theta) is an hyper-parameter of the inner objective.
+# but l2reg = jnp.exp(theta) is an hyper-parameter of the inner objective.
 def outer_objective(theta, init_inner, data):
   """Validation loss."""
   X_tr, X_val, y_tr, y_val = data
-  # We use the bijective mapping lam = jnp.exp(theta) to ensure positivity.
-  lam = jnp.exp(theta)
-  w_fit = ridge_solver(init_inner, lam, (X_tr, y_tr))
+  # We use the bijective mapping l2reg = jnp.exp(theta) to ensure positivity.
```

Comment:
suggestion "to ensure positive" -> "to optimize in log-space and ensure positivity." (I believe both are important)